packages/main/src/resolvers/conversation.test.ts

```diff
@@ -638,7 +638,7 @@ describe("when addressing conversations", () => {
     })
   })
 
-  it("an edited message will have an isRead field", async () => {
+  it("marks an edited message as read", async () => {
```

Comment:
This title and the test itself are much improved! But I take issue with the word "marks" - that implies a state change, but what you are testing is an interpretation of some given data. I would say "interprets an edited message..." or "treats an edited message as...". But that still does not accurately describe what you are testing. It is possible for an edited message to be unread. This test specifies that the read status of a presentable is based on the read status of the most recent revision.